src/elements/README.md

```diff
@@ -13,7 +13,7 @@ The `messages` property is a map of message keys and translated strings. All the
 If you are using the CDNs, the i18n messages are included in the bundle.
 
 Example of using the `language` and `messages` properties:
-```jsx
+```jsx static
```

Comment:
qq: what's this change do?

Reply:
prevents react-styleguidist from rendering the code, this is presenting the code as an example not meant to be rendered